### PR TITLE
 Function and file names different in python modules

### PR DESCRIPTION
**Pull request by giovanni-mocellin**

### CODE REVIEW

**Comment by jshlee** [comment]
FYI, u dont need to close PR
u can just push to the same branch and the original PR get updated

File: Validation/GEMCosmicMuonStand/test/StandGeometryConfiguration_run1.csv [hunk 1]
```diff
@@ -1,15 +1,15 @@
 RunNumber,1,
```
Comment (jshlee) [inline Validation/GEMCosmicMuonStand/test/StandGeometryConfiguration_run1.csv:1]: i would rather these settings goto /data dir

File: Validation/GEMCosmicMuonStand/test/excel_to_csv.py [hunk 2]
```diff
@@ -2,7 +2,7 @@
 import csv
```
Comment (jshlee) [inline Validation/GEMCosmicMuonStand/test/excel_to_csv.py:2]: move to python

File: Validation/GEMCosmicMuonStand/test/StandGeometryConfiguration_run1.csv (hunk 1 above)
Comment (giovanni-mocellin) [reply Validation/GEMCosmicMuonStand/test/StandGeometryConfiguration_run1.csv:1]: Do you mean to place the gem11L_c1_r1.xml and other xml files in the folder gemcrs/src/Geometry/MuonCommonData/data/ ?

File: Validation/GEMCosmicMuonStand/test/excel_to_csv.py (hunk 2 above)
Comment (giovanni-mocellin) [reply Validation/GEMCosmicMuonStand/test/excel_to_csv.py:2]: Same for all the others python modules?

File: Validation/GEMCosmicMuonStand/test/StandGeometryConfiguration_run1.csv (hunk 1 above)
Comment (jshlee) [reply Validation/GEMCosmicMuonStand/test/StandGeometryConfiguration_run1.csv:1]: Validation/GEMCosmicMuonStand/data

File: Validation/GEMCosmicMuonStand/test/excel_to_csv.py (hunk 2 above)
Comment (jshlee) [reply Validation/GEMCosmicMuonStand/test/excel_to_csv.py:2]: yes
but not to python run files

**Comment by giovanni-mocellin** [comment]
I've pushed a new commit to my branch, but it is not showing it here...

**Comment by giovanni-mocellin** [comment]
I have done all the corrections requested

**Comment by jshlee** [comment]
u need to push to github

**Pull request merged**